tools/topology/topology2/include/controls/bytes.conf

```diff
@@ -0,0 +1,72 @@
+<include/controls/common.conf>
+
```

Comment:
Can you include an example instantiation in the comments here like the others have.

Reply:
sure